source/MaterialXGenShader/ShaderNode.h

```diff
@@ -144,6 +144,9 @@ class ShaderInput : public ShaderPort
     /// Get optional channels value
     const string& getChannels() const { return _channels; }
 
+    /// Flag set on input shader port.
+    static const unsigned int BIND_INPUT = 1 << 1;
```

Comment:
To make it easier for readers to track the list of supported flags, I would recommend moving this definition just below the definition of ShaderPort::EMITTED:

```
    /// Flags set on shader ports.
    static const unsigned int EMITTED = 1 << 0;
    static const unsigned int BIND_INPUT = 1 << 1;
```